src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate.UnitTests/QueryTests.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Wox.Infrastructure;
+using Wox.Plugin;
+
+namespace Microsoft.PowerToys.Run.Plugin.TimeDate.UnitTests
+{
+    [TestClass]
+    public class QueryTests
+    {
+        [TestInitialize]
+        public void Setup()
+        {
+            StringMatcher.Instance = new StringMatcher();
+        }
+
+        [DataTestMethod]
+        [DataRow("time", 12)]
+        [DataRow("date", 11)]
+        [DataRow("year", 4)]
```

Comment:
should not be true. to many results 

setting problem (bool.default == false) => Main.UnitTestForceGlobalFormatSettingTrue